content/articles/2018-02-devtools-refactor.Rmd

```diff
@@ -0,0 +1,65 @@
+---
+title: The refactoring of devtools
+slug: devtools-refactor
+description: >
+  Refactoring devtools: why and how
+author: Jenny Bryan
+date: 2018-02-26
+categories:
+  - package
+tags: []
+photo:
+  url: https://morguefile.com/p/734816
+  author: mconnors
+---
+
+```{r setup, include = FALSE}
+library(usethis)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+[devtools](https://github.com/r-lib/devtools#readme) is an R package that helps you create, develop, and install R packages. By 2017, it was clear that devtools would be easier to maintain if it was split into a few smaller and more focused packages. For example, there is internal tension from trying to serve two radically opposing aims: a batteries-included framework for rapid iteration in local package development and a minimal-dependency package installer and checker, e.g., for continuous integration settings like [Travis CI](https://travis-ci.org).
+
+[usethis](http://usethis.r-lib.org/) is the first of these more focused packages to be released publicly on CRAN. usethis is the home for functions related to package and project setup. Writing the blog post for its recent CRAN release (ADD LINK) inspired us to clarify the devtools situation in this post.
+
+Other packages in development on GitHub, but not yet released on CRAN:
+
+  * FILL
+  * THIS
+  * IN?
+
+## Long live devtools!
+
+devtools has a bright future and is an important package for the r-lib/tidyverse team. devtools will continue to exist as a meta-package, exposing functionality from lower-level, domain-specific packages. The implementation will be different from the [tidyverse meta-package](http://tidyverse.tidyverse.org), but the spirit is the same.
```

Comment:
If we want to shift away from having r-lib and tidyverse as synonymous (at least in an outward facing way), I would change team ⇨ _teams_.